### PR TITLE
Add password rule defaults for new installations of silverstripe

### PR DESCRIPTION
**Pull request by flamerohr**
For more details: https://github.com/silverstripe/silverstripe-framework/issues/7724

### CODE REVIEW

File: mysite/_config.php [hunk 1]
```diff
@@ -1 +1,11 @@
...
+
+use SilverStripe\Security\PasswordValidator;
+use SilverStripe\Security\Member;
+
+// remove PasswordValidator for SilverStripe 5.0
+$validator = new PasswordValidator();
+
+$validator->minLength(8);
+$validator->checkHistoricalPasswords(6);
+Member::set_password_validator($validator);
```
Comment (robbieaverill) [inline mysite/_config.php]: If we add this into a call_user_func callback it'll avoid creating global variables (as far as I know)

**Comment by robbieaverill** [comment]
Yay, this means we can remove some of this too: https://github.com/silverstripe/cwp-core/blob/master/_config.php#L30-L36

File: mysite/_config.php (hunk 1 above)
Comment (flamerohr) [reply mysite/_config.php]: global variables has been fixed in an earlier pull request by @kinglozzer :)
so it's not longer an issue for this case

Comment (flamerohr) [reply mysite/_config.php]: Here's a link: https://github.com/silverstripe/silverstripe-framework/pull/7493

**Pull request merged**